src/operator/slice_channel-inl.h

```diff
@@ -62,11 +67,63 @@ struct SliceChannelParam : public dmlc::Parameter<SliceChannelParam> {
   }
 };  // struct SliceChannelParam
 
+struct SplitKernel {
+  template<typename DType>
+  static MSHADOW_XINLINE void Map(size_t i,
```

Comment:
Could you please add a comment on the Map operator that helps understand the approach of the implementation? that would be helpful for future maintainers.

Reply:
Added